cdap-proto/src/main/java/io/cdap/cdap/proto/artifact/ChangeSummary.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2014-2022 Cask Data, Inc.
```

Comment:
new files should have copyright year of when they were created. Just 2022 instead of 2014-2022

Reply:
still have this comment